src/registrar/utility/csv_export.py

```diff
@@ -1,44 +1,90 @@
 import csv
+import logging
+from datetime import datetime
 from registrar.models.domain import Domain
 from registrar.models.domain_information import DomainInformation
 from registrar.models.public_contact import PublicContact
 from django.db.models import Value
 from django.db.models.functions import Coalesce
+from itertools import chain
+from django.utils import timezone
 
+logger = logging.getLogger(__name__)
 
-def export_domains_to_writer(writer, columns, sort_fields, filter_condition):
+
+def get_domain_infos(filter_condition, sort_fields):
+    domain_infos = DomainInformation.objects.filter(**filter_condition).order_by(*sort_fields)
+    return domain_infos
+
+
+def write_row(writer, columns, domain_info):
+    security_contacts = domain_info.domain.contacts.filter(contact_type=PublicContact.ContactTypeChoices.SECURITY)
+    # For linter
+    ao = " "
+    if domain_info.authorizing_official:
+        first_name = domain_info.authorizing_official.first_name or ""
+        last_name = domain_info.authorizing_official.last_name or ""
+        ao = first_name + " " + last_name
+    # create a dictionary of fields which can be included in output
+    FIELDS = {
+        "Domain name": domain_info.domain.name,
+        "Domain type": domain_info.get_organization_type_display() + " - " + domain_info.get_federal_type_display()
+        if domain_info.federal_type
+        else domain_info.get_organization_type_display(),
+        "Agency": domain_info.federal_agency,
+        "Organization name": domain_info.organization_name,
+        "City": domain_info.city,
+        "State": domain_info.state_territory,
+        "AO": ao,
+        "AO email": domain_info.authorizing_official.email if domain_info.authorizing_official else " ",
+        "Security contact email": security_contacts[0].email if security_contacts else " ",
+        "Status": domain_info.domain.get_state_display(),
+        "Expiration date": domain_info.domain.expiration_date,
+        "Created at": domain_info.domain.created_at,
+        "First ready at": domain_info.domain.first_ready_at,
+        "Deleted at": domain_info.domain.deleted_at,
+    }
+    writer.writerow([FIELDS.get(column, "") for column in columns])
+
+
+def export_domains_to_writer(
```

Comment:
Rather than add two new parameters to this function, can this function just be called twice, with the different parameters?
It would require removing the writing of column headers to another function.

Reply:
Done. Good suggestion.